src/main/java/com/example/petbuddybackend/entity/user/Caretaker.java

```diff
@@ -45,12 +45,16 @@ public class Caretaker {
     private Integer numberOfRatings;
 
     @Basic(fetch = FetchType.LAZY)
-    @Formula("(SELECT AVG(r.rating) " +
+    @Formula("(SELECT COALESCE(AVG(r.rating), 0.0) " +
             " FROM Rating r " +
             " JOIN Care c ON r.care_id = c.id " +
             " WHERE c.caretaker_email = email)")
     private Float avgRating;
 
+    @Column(nullable = false)
+    @Builder.Default
+    private Float ratingScore = 0.0f;
```

Comment:
Skoro to jest `nullable=false`, to ten update score'a przejdzie jak ktoś nie ma ocen? Bo nie wiem czy nulla nie będzie chciał przypisać

Reply:
No chyba że faktycznie przypisać avgRating na 0 ale jak pisałem, to powinno móc być nullem